airflow/models/baseoperator.py

```diff
@@ -1826,11 +1753,47 @@ def get_serialized_fields(cls):
                     'operator_extra_links',
                     'upstream_task_ids',
                     'task_type',
+                    # These are automatically populated from partial_kwargs. In
+                    # a perfect world, they should be properties like other
+                    # partial_kwargs-populated values e.g. 'queue' below, but we
+                    # must match BaseOperator's implementation and declare them
+                    # as writable attributes instead.
```

Comment:
We _could_ choose to make them writable properties that affect the slot in partial_kwargs couldn't we?

```python
    @weight_rule.setter
    def _set_weigh_rule(self, value):
        self.partial_kwargs['weight_rule'] = value
```

Reply:
Hmm yeah theoratically (and maybe also a deleter for Mypy). I’ll investigate.

Reply:
Uh, Mypy is not smart enough unfortunately python/mypy#4125

Reply:
Type ignore it is the other option?